File: bin/Debug/net5.0/v4.4.2/lgcy-on-lgcy/examples/s_api-captcha-react-webapi2/csharp/Backend/Models/CaptchaModel.cs
using Newtonsoft.Json;

namespace ReactWebAPICaptchaExampleCSharp.Backend.Models
{
    public class CaptchaBasicModel
    {
        [JsonProperty("captchaId")]
        public string CaptchaId { get; set; }

        [JsonProperty("userEnteredCaptchaCode")]
        public string UserEnteredCaptchaCode { get; set; }
    }

    public class CaptchaContactModel
    {
        [JsonProperty("name")]
        public string Name { get; set; }

        [JsonProperty("email")]
        public string Email { get; set; }

        [JsonProperty("subject")]
        public string Subject { get; set; }

        [JsonProperty("message")]
        public string Message { get; set; }

        [JsonProperty("captchaId")]
        public string CaptchaId { get; set; }

        [JsonProperty("userEnteredCaptchaCode")]
        public string UserEnteredCaptchaCode { get; set; }

    }
}

File: bin/Debug/net5.0/v4.4.2/lgcy-on-lgcy/examples/s_api-captcha-jquery-webapi2/vbnet/Backend/Models/CaptchaModel.vb
Imports Newtonsoft.Json

Public Class CaptchaBasicModel

    <JsonProperty("captchaId")>
    Public Property CaptchaId As String

    <JsonProperty("userEnteredCaptchaCode")>
    Public Property UserEnteredCaptchaCode As String
End Class

Public Class CaptchaContactModel

    <JsonProperty("name")>
    Public Property Name As String

    <JsonProperty("email")>
    Public Property Email As String

    <JsonProperty("subject")>
    Public Property Subject As String

    <JsonProperty("message")>
    Public Property Message As String

    <JsonProperty("captchaId")>
    Public Property CaptchaId As String

    <JsonProperty("userEnteredCaptchaCode")>
    Public Property UserEnteredCaptchaCode As String
End Class
